h2o-cve-2018-0608/README.md

```diff
@@ -0,0 +1,43 @@
+# h2o memory corruption CVE example
+
+This repo replicates finding a memory corruption bug inside h2o with fuzzing.
+
+> Note: since this finds the bug in an unmodified h2o binary
+> (a *network target*), it can only be found by fuzzers that support network
+> fuzzing (such as Mayhem).
+
+## To build
+
+Assuming you just want to build the docker image, run from the project
+directory (`h2o-cve-2018-0608`):
+
+```bash
+docker build -t forallsecure/h2o-cve-2018-0608 .
+```
+
+## Get from Dockerhub
+
+If you don't want to build locally, you can pull a pre-built image
+directly from dockerhub:
+
+```bash
+docker pull forallsecure/h2o-cve-2018-0608
+```
+
+
+## Run under Mayhem
+
+From the project directory (`h2o-cve-2018-0608`) run:
+
+```bash
+mayhem run mayhem/h2o
+```
+
+## POC
+
+We have included a proof of concept output under the `poc`
+directory.
+
+Note: Fuzzing has some degree of non-determinism, so when you run
+yourself you may not get exactly this file.  This is expected; your
+output should still trigger the memory corruption bug.
```

Comment:
Pls add how long it takes us to find. Such as:
On our test cluster, given the original seed, it takes about xxx minutes to reproduce the bug.